### PR TITLE
New awk Cheat Sheet

### PR DESCRIPTION
**Pull request by gamebusterz**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] New Instant Answer
  - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

New awk Cheat Sheet
###### People to notify

@pjhampton 

---

Instant Answer Page: https://duck.co/ia/view/awk_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @gamebusterz

### CODE REVIEW

**Comment by daxtheduck** [comment]
@gamebusterz Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=awk cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/awk_cheat_sheet_mobile.png?z6VTRBAdDz)

File: share/goodie/cheat_sheets/json/awk.json [hunk 1]
```diff
@@ -0,0 +1,270 @@
+{
+    "id": "awk_cheat_sheet",
+    "name": "awk Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/awk.json]: Lets change name to "AWK"

File: share/goodie/cheat_sheets/json/awk.json [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    "description": "Awk is a text processing and data extraction utility.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/awk.json]: Remove the period at end of descrptn

**Comment by pjhampton** [comment]
@jophab this PR isn't going to be merged for a while due to the programming mission (https://forum.duckduckhack.com/) so please hold off the feedback for now.

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**